lms/services/auto_grading.py

```diff
@@ -52,3 +53,21 @@ def calculate_grade(
 
     grade = min(100, grade)  # Proportional grades are capped at 100%
     return round(grade, 2)
+
+
+def get_last_synced_grade_for_h_userids(db, h_userids: list[str], assignment_id):
```

Comment:
Query all the previous grades from a list of students. Useful to display it on metrics endpoint.